### ISSUE TITLE
com.datastax.driver.core.Session are closed with the EntityManager

### ISSUE THREAD

**Issue by jfgosselin** [Impetus/kundera#632]
From the API http://www.datastax.com/drivers/java/2.0/

"Session instances are thread-safe and usually a single instance is enough per application. "

Right now  the Session is closed  after each EntityManager.close() in DSClientFactory. This isn't optimal since each session maintains multiple connections to the cluster nodes and provides policies to choose which node to use for each query (e.g. round-robin ). 

DSClientFactory.java :

```
   void releaseConnection(Session session)
    {
        if (session != null)
        {
            session.closeAsync();
        }
    }
```

**Comment by chhavigangwal** [comment]
@jfgosselin 

Agreed, will get this fixed in upcoming release so that session is closed only when EntityManagerFactory is closed.

Chhavi

**Comment by chhavigangwal** [comment]
Fixed with Kundera -2.14 Release
